### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
##  Calculator UI Design
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class-based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

Warm regards 👍🏽 
@kinginthenorthcodez

### CODE REVIEW

File: src/component/calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          {total}
+        </div>
+        <div className="keypadBtns">{items}</div>
+      </div>
+    );
+  }
+}
+
+export default KeypadUI;
+// const btn = ['AC', '+/-', '%', '÷', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', ' ='];
```
Comment (Wes-Isaac) [inline src/component/calculator.js]: - [ ] You have done a great job so far, kindly remove the commented code to keep your code clean 👍.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are set up properly ✔️.
- Great summary😲. 
- Project works as expected 😮.
- README is exquisite.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/component/calculator.js (hunk 1 above)
Comment (kinginthenorthcodez) [reply src/component/calculator.js]: Removed commented code

**Review by mateo951 — APPROVED** [review]
Hi @kinginthenorthcodez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
Merged #2  into development

**Pull request merged**